src/device/simulator.ts

```diff
@@ -114,9 +190,25 @@ export class SimulatorDeviceConnection
     });
   }
 
-  sensorWrite = async (id: string, value: number): Promise<void> => {
-    this.postMessage("sensor_set", {
-      sensor: id,
+  setSimulatorValue = async (
+    id: string,
```

Comment:
This can probably be SensorStateKey now. Nothing tracks the sensor types but not sure that's worth it.